doc/specs/#120,121,119-Update-Uninstall-List.md

```diff
@@ -0,0 +1,229 @@
+---
+author: <first-name> <last-name> <github-id>/<email>
+author: <kevin> <larkin> <kevinlams/<kevinla@microsoft.com>
+created on: <2020-06-03>
+last updated: <2020-06-04>
+issue id: <120, 121, 119>
+---
+
+# ===== DRAFT ======
+# Windows Package Manager Upgrade, Uninstall and List Command Soec
+
+## Abstract
+
+For our preview of the Windows Package Manager, the goal was to enable install of apps.  In order to be a package manager, the Windows Package Manager must be able to list all installed packages, communicate if there is an update and uninstall apps.  
+
+This spec will cover the following commands: list, update, and uninstall.
+## Inspiration
+
+See abstract.  :)
+
+## Solution Design
+
+* **List**
+* **Update \ Upgrade**
+* **Uninstall**
+
+### List
+
+The **list** command is a very specific search.  List will show the applications currently installed.  **List** will also compare the installed version to the known repository version.  If there is an update available, this will be provided as well.
+
+## Usage
+
+```winget list [[-q] <query>] [<options>]```
+
+![list command](Images/120-List.png)
+
+**List** when executed by itself will show all apps installed by the Windows Package Manager.
+
+## Arguments
+
+The following arguments are available.
+
+| Argument  | Description |
+|--------------|-------------|
+| **-q,--query** |  The query used to search for an application. |
+| **-?, --help** |  Gets additional help on this command. |
```

Comment:
Not a huge deal, but I'd personally prefer the `-h` alias as it is far more common and more clearly maps to `--help` than `-?`

Reply:
Isn't `-?` the standard on Windows while `-h` is more a Unix thing?